paper/2023-EVCS/content/introduction.tex

```diff
@@ -52,7 +52,10 @@
 highlighted by the language workbench challenge~\cite{DBLP:conf/sle/ErdwegSVBBCGHKLKMPPSSSVVVWW13} for what constitutes a language workbench.
 %
 %
-Although not all of the optional criteria for a language workbench are met by dependently typed languages, we are convinced that with some additional engineering (taking advantage of existing work, for example Quickchick~\cite{DBLP:journals/pacmpl/LampropoulosPP18}) more optional criteria can be satisfied.
+Although not all of the optional criteria for a language workbench are met by dependently typed languages, we are convinced that with some additional engineering
+%(taking advantage of existing work
+% , for example Quickchick~\cite{DBLP:journals/pacmpl/LampropoulosPP18})
+more optional criteria can be satisfied.
```

Comment:
This turns an evidence-supported sentence into a "we believe" one though.

Reply:
i think we can revert it, as we have both saved space with our respective read through...